### PR TITLE
inode utilisiation option is added to disk metrics.

### PR DESCRIPTION
**Pull request by caglar84**
From time to time we face "No space left on device" issue, although there is space on disk. The problem is directly related to 100% inode usage. So we need to monitor inode utilisation on instances too. 

"df" command is able to give block storage or inode statistics according to "-i" parameter. So I executed df command twice to get both of those data. 2 additional iterations are added to get_disk_info method. The first one works as-is by adding inode_util value as 0. The second iteration calculates inode usage and puts this value on a temp list. The third iteration aggregates two lists.

### CODE REVIEW

File: cloudwatchmon/cli/put_instance_stats.py [hunk 1]
```diff
@@ -374,7 +380,20 @@ def get_disk_info(paths):
         total = int(line[1]) * 1024
         used = int(line[2]) * 1024
         avail = int(line[3]) * 1024
-        disks.append(Disk(mount, file_system, total, used, avail))
+        disks.append(Disk(mount, file_system, total, used, avail, 0))
+
+    df_inode_out = [s.split() for s in
```
Comment (osiegmar) [inline cloudwatchmon/cli/put_instance_stats.py]: Could you please only gather inode usage if it was requested via CLI argument?

Comment (osiegmar) [inline cloudwatchmon/cli/put_instance_stats.py:389]: Could you please only gather inode usage if it was requested via CLI argument?

File: cloudwatchmon/cli/put_instance_stats.py [hunk 2]
```diff
@@ -295,13 +296,18 @@ def config_parser():
     disk_group.add_argument('--disk-space-avail',
                             action='store_true',
                             help='Reports available disk space in gigabytes.')
+    disk_group.add_argument('--disk-inode-util',
```
Comment (osiegmar) [inline cloudwatchmon/cli/put_instance_stats.py]: Please move this below the disk-space block (below the disk-space-units)

**Review by osiegmar — CHANGES_REQUESTED** [review]
Thanks for your work! Please check my review comments.

**Comment by caglar84** [comment]
Thanks for the review.

**Comment by osiegmar** [comment]
Thanks!

**Pull request merged**